husky_description/urdf/husky.urdf.xacro

```diff
@@ -33,6 +33,7 @@ ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSI
   <xacro:include filename="$(find husky_description)/urdf/accessories/sick_lms1xx_mount.urdf.xacro"/>
   <xacro:include filename="$(find husky_description)/urdf/accessories/sensor_arch.urdf.xacro"/>
   <xacro:include filename="$(find ur_description)/urdf/ur5.urdf.xacro" />
+  <xacro:include filename="$(find pr2_description)/urdf/gripper_v0/gripper.urdf.xacro"/>
```

Comment:
Are you actually using this macro anywhere? If so, pr2_description needs to be added as a run_depend to this package.